README.md

```diff
@@ -1,207 +1,9 @@
-# Cypress Webpack Preprocessor [![CircleCI](https://circleci.com/gh/cypress-io/cypress-webpack-preprocessor.svg?style=svg)](https://circleci.com/gh/cypress-io/cypress-webpack-preprocessor) [![semantic-release][semantic-image] ][semantic-url]
+# Cypress Webpack Preprocessor
 
 Cypress preprocessor for bundling JavaScript via webpack
 
-## Installation
+# :warning: THIS REPO HAS BEEN ARCHIVED :warning:
```

Comment:
```suggestion
# :warning: THIS REPO HAS BEEN MOVED :warning:
```

At first glance, "archived" sounds like we're killing off this project, so I think we should say it's been "moved".